domains/client/domain-operator/src/bundle_processor.rs

```diff
@@ -20,6 +20,11 @@ use sp_messenger::MessengerApi;
 use sp_runtime::traits::{Block as BlockT, Zero};
 use sp_runtime::{Digest, DigestItem};
 use std::sync::Arc;
+use std::time::Instant;
+
+// The slow log threshold, 2000ms of block execution (from the maximum block weight)
```

Comment:
Domain block execution time may well exceed 2 sec time since we do not have any limit. IMO, this is too strict and when the domain blocks are heavy, which would be very often, logs will be have this log line almost always. 

Do we actually need to print such message for execution and instead we can print slowness for just consensus block preprocessing since that should not be slow.

Any thoughts?
cc: @nazar-pc

Reply:
Actually, there is a limit on the domain block execution time, which is enforced by the `CheckWeight` SignedExtension, and it is set to 2000ms:
https://github.com/subspace/subspace/blob/165c319ca4f6e178b7d09c401386205cacb8f443/domains/runtime/evm/src/lib.rs#L205-L206
If a domain block includes more extrinsic than this limit, the following extrinsic will be skipped from execution once the weight limit is hit.

>Do we actually need to print such message for execution and instead we can print slowness for just consensus block preprocessing since that should not be slow.

The slow log covers the whole process of domain block derivation that is more end-to-end and what we care IMO.

Reply:
Or perhaps we can add more fine grained slow logs, one for consensus block preprocessing and one for domain block execution based on the block's actual weight instead of the max block weight.

Reply:
> Or perhaps we can add more fine grained slow logs, one for consensus block preprocessing and one for domain block execution based on the block's actual weight instead of the max block weight.

That would be Ideal IMO

As for the `WEIGHT_MILLISECS_PER_BLOCK` being 2000ms, I think we should revisit this since we cannot meet such weight limit with EVM contracts in general.